src/components/generic/Button/Button.scss

```diff
@@ -90,3 +90,26 @@
     color: var(--wxc-button--ghost--disabled--color);
   }
 }
+
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-button--start {
+  font-size: 0.938rem;
+  line-height: 1.25rem;
+  padding: 0.25rem 5.719rem;
+  letter-spacing: 0.015px;
```

Comment:
Do we need letter-spacing here, is this css property used in figma?

Reply:
yes